end_to_end/test_gemma.sh

```diff
@@ -1,19 +1,28 @@
 #!/bin/bash
 set -ex
 idx=$(date +%Y-%m-%d-%H-%M)
-# convert 2.5B checkpoint
-export base_model_path=gs://maxtext-gemma/flax/2b
-export maxtext_model_path=gs://maxtext-gemma/2b/${idx}
-python MaxText/convert_gemma_chkpt.py --base_model_path ${base_model_path} --maxtext_model_path ${maxtext_model_path} --model_size 2b
+# Download checkpoint weights and copy them to GCS bucket at $CHKPT_BUCKET \
```

Comment:
"If you want to reproduce these results"

Reply:
Or maybe between is:

"This is an automated test that the MaxText team ensures continues to work. To run it on your own hardware, we recommend:" and then put recommendations.